fm.reaper.Reaper.yml

```diff
@@ -0,0 +1,70 @@
+app-id: fm.reaper.Reaper
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+tags:
+ - proprietary
+command: /app/REAPER/reaper
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  - --device=all
+  - --filesystem=home
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+  - --system-talk-name=org.freedesktop.Flatpak
+  - --env=ALSA_CONFIG_PATH=
+  - --env=LV2_PATH=/app/extensions/Plugins/lv2
+  - --env=DSSI_PATH=/app/extensions/Plugins/dssi
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
```

Comment:
does Reaper really support these plugins formats?

Reply:
I wanted to leave the option of the types of plugins that the user could use. Too much?

Reply:
what option? Does Reaper support them?